src/middlewares/validators/updateCategoryPayload.js

```diff
@@ -0,0 +1,16 @@
+const Joi = require('joi');
+
+const updateCategoryPayloadValidator = (req, res, next) => {
+  const schema = Joi.object({
+    name: Joi.string().min(2).max(30).required()
```

Comment:
It there any difference between create and update category payload?

Reply:
Fixed.